src/shell/components/load-instance/index.js

```diff
@@ -60,7 +63,8 @@ export default connect(state => {
               props.products &&
               props.instance.ID &&
               props.instance.domains &&
-              props.user.ID
+              props.user.ID &&
+              props.languages.length
```

Comment:
With non multi-lang instances do we get a single default entry of "en-US"? Making this case work for both types of instances?